vaadin-grid-pro-flow-integration-tests/src/main/java/com/vaadin/flow/component/gridpro/examples/MainView.java

```diff
@@ -36,6 +39,14 @@ protected void createEditorColumns() {
             subPropertyDisplayPanel.setText(newValue);
         }).setHeader("Name").setWidth("300px");
 
+        ComboBox<Department> cb = new ComboBox<>();
```

Comment:
Add a demo as well

Reply:
Demo added, but without `comboBox` to avoid extra dependency.